backend/src/email/services/email-transactional.service.ts

```diff
@@ -58,11 +68,11 @@ async function sendMessage({
       )
     : undefined
 
-  const blacklisted = await isBlacklisted(recipient)
+  const blacklisted = await EmailService.isRecipientBlacklisted(recipient)
```

Comment:
I moved this from the utils folder into `EmailService` — not 100% sure whether this is correct, hence flagging this.